va/va.h

```diff
@@ -794,6 +811,28 @@ typedef enum
      *  attribute value is \c VAConfigAttribValContextPriority
      */
     VAConfigAttribContextPriority       = 41,
+    /**
+     * \brief VAConfigAttribDisplayLinkHdcpWired
+     *
+     * This attribute specifies wired HDCP settings for protected display link.
```

Comment:
need to describe the definition of this attribute, useing \c to refer the definition of a structure , or, declare the meaning of the value.

Reply:
Remove this config attribute since it will not be used.